contribute/tool_resource_update.md

```diff
@@ -0,0 +1,49 @@
+
+The [all_tools_and_resources](/all_tools_and_resources) list is based on the [csv file](https://github.com/elixir-europe/infectious-diseases-toolkit/blob/main/_data/main_tool_and_resource_list.csv) in the `_data` directory of the Infectious Diseases Toolkit repository. Tools and resources can be manually linked to [FAIRsharing.org](https://fairsharing.org/), [Bio.tools](https://bio.tools) and [TeSS](https://tess.elixir-europe.org/), but every week we also run a fully automatic check that links tools and resources with the corresponding registries. A GitHub Bot will generate a Pull Request (PR) with the new links added to the main data file of the website (a yaml file).
+
+{% include callout.html type="important" content="The link with FAIRsharing,TeSS and Bio.tools is automatically done using GitHub actions and is weekly updated. These automatic links are not seen in the table. The search query to one of these registries for a tool or resource can be overwritten in the registry column of the main csv tool table. If no FAIRsharing ID, Bio.tools ID or TeSS Query is available for a source, but there is yet one automatically given (faulty), you can overwrite the automatic linking by adding 'NA' as registry." %}
+
+## The main table
+
+
+The main table is based on [this google spreadsheet](https://docs.google.com/spreadsheets/d/13tqfSbgivokfEkxGXPRFShVhCmO4VuTQRe4uQgJOMbc).
+
+The table consists of 5 columns:
+- **name**: the name of the tool or resource
+- **url**: URL to the main page of the tool or resource, make sure to let the URL start with `https://`
+- **description**: A short description of the tool or resource. Try to not use the characters `"` or `'` 
+- **registry**: 3 registries are supported: [Bio.tools](https://bio.tools), [FAIRsharing.org](https://fairsharing.org/) and [TeSS](https://tess.elixir-europe.org/). The keywords you can use respectively are: `biotools`, `fairsharing`, `fairsharing-coll` and `tess`, specifying the id or query with a colon). FAIRsharing collections have an ID that follows the pattern `bsg-s000XXX`. List multiple registries using a comma `, ` between the keywords to separate the key:value pairs. The values that are given in the table will always overrule the automatic links. If no FAIRsharing ID, Bio.tools ID or TeSS Query is available for a source, you can overwrite the automatic linking by adding 'NA' as registry.
+- **related_pages**: This is used to tag the tools so it is listed on the correct page. We only allow page_id that are linked to a page. To find out what the page_id of a page is, please check its metadata attribute `page_id` at the top of the markdown file or the [Website overview](website_overview) page. List multiple page_id's by using a comma `, ` between them.
```

Comment:
```suggestion
- **related_pages**: This is used to tag the tools so that they can be listed on the correct page. We only allow page_id that are linked to a page. To find out what the page_id of a page is, please check its metadata attribute `page_id` at the top of the markdown file or the [Website overview](website_overview) page. List multiple page_id's by using a comma `, ` between them.
```